test/classes/assignments/noAssignClass.good

```diff
@@ -20,5 +22,7 @@ noAssignClass.chpl:161: error: Can't overload assignments for class types
 noAssignClass.chpl:165: error: Can't overload assignments for class types
 noAssignClass.chpl:169: error: Can't overload assignments for class types
 noAssignClass.chpl:173: error: Can't overload assignments for class types
+noAssignClass.chpl:173: error: Can't overload assignments for class types
 noAssignClass.chpl:177: error: Can't overload assignments for class types
 noAssignClass.chpl:181: error: Can't overload assignments for class types
+noAssignClass.chpl:181: error: Can't overload assignments for class types
```

Comment:
What aspect of this PR resulted in getting duplications of these errors?  (it seems unfortunate).